### PR TITLE
allow json_schemer 2.3.x

### PR DESCRIPTION
**Pull request by klausmeyer**

### CODE REVIEW

File: spec/openapi_contracts/validators/request_body_spec.rb [hunk 1]
```diff
@@ -28,9 +28,19 @@
       }
     end
 
+    let(:error_disallowed_additional_property) do
```
Comment (mkon) [inline spec/openapi_contracts/validators/request_body_spec.rb:31]: 👍

**Comment by klausmeyer** [comment]
Would be nice if you could also publish a new release whenever you have time (no hurry).

**Comment by mkon** [comment]
Done :)

**Comment by klausmeyer** [comment]
:sunglasses: Awesome. Thanks a lot!

**Pull request merged**